packages/x-components/src/design-system/components/suggestion-group/default.scss

```diff
@@ -27,6 +28,17 @@
   gap: var(--x-size-gap-suggestion-group-default);
   @include safari-gap(var(--x-size-gap-suggestion-group-default));
 
+  // border
+  border: none;
+  border-block-start-width: var(--x-size-border-width-top-suggestion-group-default);
+  border-block-end-width: var(--x-size-border-width-bottom-suggestion-group-default);
+  border-inline-start-width: var(--x-size-border-width-left-suggestion-group-default);
+  border-inline-end-width: var(--x-size-border-width-right-suggestion-group-default);
+  border-radius: var(--x-size-border-radius-top-left-suggestion-group-default)
+  var(--x-size-border-radius-top-right-suggestion-group-default)
+  var(--x-size-border-radius-bottom-right-suggestion-group-default)
+  var(--x-size-border-radius-bottom-left-suggestion-group-default);
+
   .x-suggestion {
     // layout
     padding: 0;
```

Comment:
here is where the `border: none` must be, to disable the `x-suggestion` border when it is inside a `x-suggestion-group`